container-engine-lib/lib/backend_impls/kubernetes/kubernetes_manager/kubernetes_manager.go

```diff
@@ -1742,8 +1743,8 @@ func (manager *KubernetesManager) waitForPodTermination(ctx context.Context, nam
 	for time.Now().Before(deadline) {
 		pod, err := manager.GetPod(ctx, namespaceName, podName)
 		if err != nil {
-			// The pod info is not always available after deletion so we handle that gracefully
-			logrus.Warnf("An error occured trying to retrieve the just-deleted pod '%v': %v", podName, err)
+			// The pod info is not always available after deletion, so we handle that gracefully
+			logrus.Debugf("An error occured trying to retrieve the just-deleted pod '%v': %v, for checking if it was succesfully terminated; but we can't ignore this error beucase the pod info is not always available after deletion", podName, err)
```

Comment:
```suggestion
			logrus.Debugf("An error occurred trying to retrieve the just-deleted pod '%v': %v, for checking if it was successfully terminated; but we can ignore this error because the pod info is not always available after deletion", podName, err)
```

Reply:
thanks!! @laurentluce !!